### PR TITLE
add fazer um dinheirinho

### PR DESCRIPTION
**Pull request by bsgama**
@casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2614)

### CODE REVIEW

File: portugues/colaborando.md [hunk 1]
```diff
@@ -27,4 +27,8 @@ O Git te parece ser util, mas como você ainda não vive no século XXI, você e
 
 
 
-Mas te bateu uma fome danada e você não tem nada na geladeira e nenhum tostão.
\ No newline at end of file
+Mas te bateu uma fome danada e você não tem nada na geladeira e nenhum tostão.
+
+Vende seu PC?
+[SIM](venda/sim.md)
+[NÃO](venda/nao.md)
```
Comment (casey-collab) [inline portugues/colaborando.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: portugues/venda/nao.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Boa idéia. Melhor morrer de fome do que perder o lançamento de Wolfenstein 3D ano que vem.
```
Comment (casey-collab) [inline portugues/venda/nao.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: portugues/venda/sim.md [hunk 3]
```diff
@@ -0,0 +1 @@
+A sua mãe que detesta jogos de violência, mas mora na baixada fluminense, fica feliz com a sua decisão.
```
Comment (casey-collab) [inline portugues/venda/sim.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by bsgama** [comment]
@casey-collab

**Comment by casey-collab** [comment]
Looks great! :dog:

**Pull request merged**